.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -164,6 +152,16 @@ jobs:
         python3-pip
         valgrind
         ${{ matrix.packages-extra }}
+
+    # hack around https://bugs.launchpad.net/ubuntu/+source/qtchooser/+bug/1964763
+    - name: "[Ubuntu] Make qtchooser work with Qt6"
+      if: runner.os == 'Linux' && matrix.qt_version == 6
+      run: >-
```

Comment:
another potential workaround is adding `/usr/lib/qt6/bin` to `PATH`, but i'm also in discussions with upstream so lets see where that goes first...

Reply:
Could you link to that discussion?

Reply:
Adding `/usr/lib/qt6/bin` to `$PATH` instead of hacking qtchooser works too.

Reply:
Has there been any progress in those discussions upstream?